compiler/test/dotty/tools/dotc/reporting/ErrorMessagesTests.scala

```diff
@@ -42,4 +42,47 @@ class ErrorMessagesTests extends ErrorMessagesTest {
       // is a type reference to `java.lang.String`
       assert(expected.dealias =:= defn.StringType, s"expected was: $expected")
     }
+
+  @Test def overridesNothing =
+    checkMessagesAfter("refchecks") {
+      """
+        |object Foo {
+        |  override def bar: Unit = {}
+        |}
+      """.stripMargin
+    }
+    .expect { (ictx, messages) =>
+      implicit val ctx: Context = ictx
+      val defn = ictx.definitions
+
+      assertMessageCount(1, messages)
+      val OverridesNothing(member) :: Nil = messages
+      assertEquals("bar", member.name.toString)
```

Comment:
Doesn't really matter in this case, but show automatically decodes "weird" names - so let's set a good precedence :+1: